pkg/services/program/program.go

```diff
@@ -36,9 +36,9 @@ type (
 	}
 )
 
-// NewProgramService creates a ProgramService according to the given key.
-// If nil is passed, the ProgramService will be created with the default programServiceRuntime.
-func NewProgramService(programServiceKey string, runtime programServiceRuntime) (ProgramService, error) {
+// NewProgramService creates a Service according to the given key.
+// If nil is passed, the Service will be created with the default programServiceRuntime.
+func NewProgramService(programServiceKey string, runtime programServiceRuntime) (Service, error) {
```

Comment:
We could rename these three names like I suggested for the file package, check my previous comment.